dsl/src/main/kotlin/com/chutneytesting/kotlin/execution/ExecutionService.kt

```diff
@@ -37,7 +33,7 @@ class ExecutionService(
             .executeAsync(
                 ExecutionRequestDto(
                     ExecutionRequestMapper.mapScenarioToExecutionRequest(scenario, environment),
-                    environment.name,
+                    com.chutneytesting.engine.api.execution.EnvironmentDto(environment.name, environment.variables),
```

Comment:
import com.chutneytesting.engine.api.execution.EnvironmentDto; rather than inline import

Reply:
there is two EnvironmntDto used in this file (one from engine and other from environment). So I have to explicit package.